src/coffee/services/authentication.coffee

```diff
@@ -0,0 +1,16 @@
+BaseService = require './base'
+
+# Login service as defined in http://dev.sphere.io/http-api-projects-customers.html#authenticate-customer
+#
+# Example
+#   service = client.authentication
+#   service.save
```

Comment:
I really like this solution. I changed our code to use this idea while we get the idea you proposed merged!